torchvision/utils.py

```diff
@@ -51,7 +51,8 @@ def make_grid(tensor, nrow=8, padding=2,
 
         def norm_ip(img, min, max):
             img.clamp_(min=min, max=max)
-            img.add_(-min).div_(max - min)
+            if min != max:
+                img.add_(-min).div_(max - min)
```

Comment:
I think we still want to have the `img.add_(-min)` just avoid the div by 0 if `min==max`

Reply:
We could have something as simple as `img.add_(-min).div_(max - min + 1e-10)` (or any suitable small number).

Reply:
Both options are fine for me